client/src/helpers/fhir/api/CreatePatient.ts

```diff
@@ -72,22 +72,23 @@ type CreatePatientFetusResponse = {
 
 export const createPatientFetus = async (patient: Patient) : Promise<CreatePatientFetusResponse> => {
   const patientFetus = JSON.parse(JSON.stringify(patient)) as Patient;
+  const patientParent = JSON.parse(JSON.stringify(patient)) as Patient;
   patientFetus.id = undefined;
 
   patientFetus.extension.find((ext) => ext.url === EXTENSION_IS_PROBAND)!.valueBoolean = true;
   patientFetus.extension.find((ext) => ext.url === EXTENSION_IS_FETUS)!.valueBoolean = true;
 
-  patient.gender = FEMALE_GENDER;
+  patientParent.gender = FEMALE_GENDER;
 
   const familyGroup = new FamilyGroupBuilder()
     .withType('person')
     .withStructure(FamilyStructure.Duo)
     .withActual(true)
     .build();
 
-  const isNewPatient = patient.id == null;
+  const isNewPatient = patientParent.id == null;
```

Comment:
personnaly im not a fan of == since it does not check the type of the varriable. For this I prefer use isEmpty from lodash which check for all sort of 'null' typing (0,'',null,undefined,{},[]);

Reply:
I'm usually in the "no `==`" team too but it's something that is used often in the project so I keep the same "coding style". And I'm simply renaming the variable here